plexus-compiler-its/pom.xml

```diff
@@ -12,9 +12,11 @@
   <packaging>pom</packaging>
 
   <name>Plexus Compiler It Tests</name>
-  
+
   <properties>
     <junit.version>4.13.2</junit.version>
+    <maven.compiler.source>11</maven.compiler.source>
```

Comment:
why not using `release`?

Reply:
netbeans would show this as warning, assuming it can figure out the compiler plugin version :)

Reply:
PR #277
Feel free to open or with better solution

Reply:
Feel free to improve opening pr